nightly/nightly.txt

```diff
@@ -138,7 +138,7 @@ expensive --timeout=600 near-chain gc tests::test_gc_star_large
 
 # lib tests
 lib near-chunks test::test_seal_removal
-lib near-chain store::tests::test_clear_old_data_too_many_heights
+lib --timeout=200 near-chain store::tests::test_clear_old_data_too_many_heights
```

Comment:
why not give a bit more time like 300?

Reply:
I gave twice of how long it's executing. Should we give more?

Reply:
Updated.